code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -99,20 +99,21 @@ There are several things that need to be remembered:
 #define SUIT_STORE_LAYER 13
 #define BACK_LAYER       14
 #define HAIR_LAYER       15
-#define L_EAR_LAYER      16
-#define R_EAR_LAYER      17
-#define FACEMASK_LAYER   18
-#define HEAD_LAYER       19
-#define COLLAR_LAYER     20
-#define HANDCUFF_LAYER   21
-#define LEGCUFF_LAYER    22
-#define L_HAND_LAYER     23
-#define R_HAND_LAYER     24
-#define FIRE_LAYER       25		//If you're on fire
-#define TARGETED_LAYER   26		//Layer for the target overlay from weapon targeting system
-#define TOTAL_LAYERS     26
+#define EARS_LAYER       16
+#define FACEMASK_LAYER   17
+#define HEAD_LAYER       18
+#define COLLAR_LAYER     19
+#define HANDCUFF_LAYER   20
+#define LEGCUFF_LAYER    21
+#define L_HAND_LAYER     22
+#define R_HAND_LAYER     23
+#define FIRE_LAYER       24		//If you're on fire
+#define TOTAL_LAYERS     24
 //////////////////////////////////
 
+#define UNDERSCORE_OR_NULL(target) "[target ? "[target]_" : ""]"

```

Comment:
I don't like the name of this define, but I can't think of something better that isn't horrendously long.